dashboard/src/components/AppUpgrade/AppUpgrade.tsx

```diff
@@ -29,11 +29,11 @@ interface IAppUpgradeProps {
   ) => Promise<boolean>;
   clearRepo: () => any;
   checkChart: (repo: string, chartName: string) => any;
-  fetchChartVersions: (id: string) => Promise<{}>;
-  getApp: (releaseName: string, namespace: string) => Promise<void>;
-  getBindings: () => Promise<IServiceBindingWithSecret[]>;
-  getChartVersion: (id: string, chartVersion: string) => Promise<{}>;
-  getChartValues: (id: string, chartVersion: string) => Promise<any>;
+  fetchChartVersions: (id: string) => Promise<any>;
```

Comment:
shouldn't this be void also?

Reply:
`fetchChartVersions` returns a list of `IChartVersions` and it's used in `src/actions/charts.ts`

Reply:
Avoid what? The use of any? And what's the difference of returning an empty object.

To be clear, what I am trying to do here is to be sure that we use action creations to only dispatch values, not to rely on any internal returned value and use them outside of the redux scope. 

In this specific case though I believe that I could not remove the returned value because as @andresmgot mentioned, this function is called from another action creator and changing it will require a bigger change.

So basically I've tried to not to return anything for actions that are pure but maintain for now the others.

In any case, I am going to double check this one, thanks for pointing it out.

Reply:
> Avoid what? The use of any? And what's the difference of returning an empty object.

I think you misread, I was asking why this isn't `void`, but I see now that the return value is being used and we don't want to change this.

Reply:
Ahhh, sorry for the misunderstanding. Yes. that's the reason.